src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/index.tsx

```diff
@@ -66,9 +85,38 @@ export const InputDisplayNameOverlay = ({
 }: InputDisplayNameOverlayProps) => {
   const intl = useIntl();
   const [alertsState, setAlertsState] = useState<string[]>([]);
+  const [isWaiting, setIsWaiting] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isWaiting, setIsWaiting] = useState(false);
```
I don't think it is needed. The type will be infer and you won't be able to use something else than boolean IMO

Reply:
ok done